packages/scheduled-tasks/README.md

```diff
@@ -0,0 +1,253 @@
+
+```typescript
+// mute command
+
+import { Command, CommandOptions, PieceContext } from '@sapphire/framework';
+import type { Message } from 'discord.js';
+
+export class MuteCommand extends Command {
+	public constructor(context: PieceContext) {
+		super(context, {
+			description: 'Mute a user'
+		});
+	}
+
+	public async run(message: Message) {
+		// create a task to unmute the user in 1 minute
+		this.container.tasks.create('unmute', { authorId: message.author.id }, 60000);
+	}
+}
+```
+
+### Create a task handler
+
+Scheduled tasks use their own store, like other types of pieces. You can create a directory alongside your commands directory named `scheduled-tasks` and place your tasks there, but they must inherit from `ScheduledTask`, like so.
+
+#### Manual task example
+
+##### Creating the Piece:
+
+```typescript
+import type { PieceContext } from '@sapphire/framework';
+import { ScheduledTask } from '@sapphire/plugin-scheduled-tasks';
+
+export class ManualTask extends ScheduledTask {
+	public constructor(context: PieceContext) {
+		super(context);
+	}
+
+	public async run(payload: unknown) {
+		this.container.logger.info('I ran!', payload);
+	}
+}
+
+declare module '@sapphire/framework' {
+	interface ScheduledTasks {
+		manual: never;
+	}
+}
+```
+
+##### Using Manual Tasks
+
+```typescript
+container.tasks.create('manual', payload, 5000);
+```
+
+#### Cron Task Example
+
+Cron jobs are currently only supported by the Redis strategy.
+
+##### Creating the Piece:
+
+```typescript
+import type { PieceContext } from '@sapphire/framework';
+import { ScheduledTask } from '@sapphire/plugin-scheduled-tasks';
+
+export class CronTask extends ScheduledTask {
+	public constructor(context: PieceContext) {
+		super(context, {
+			cron: '0 * * * *'
+		});
+	}
+
+	public async run() {
+		this.container.logger.info('I run every hour at *:00');
+	}
+}
+
+declare module '@sapphire/framework' {
+	interface ScheduledTasks {
+		cron: never;
+	}
+}
+```
+
+##### Using Cron tasks
+
+Cron & Interval tasks are loaded automatically.
+
+#### Interval task example
+
+##### Creating the Piece:
+
+```typescript
+import type { PieceContext } from '@sapphire/framework';
+import { ScheduledTask } from '@sapphire/plugin-scheduled-tasks';
+
+export class IntervalTask extends ScheduledTask {
+	public constructor(context: PieceContext) {
+		super(context, {
+			interval: 60 * 1000 // 60 seconds
+		});
+	}
+
+	public async run() {
+		this.container.logger.info('I run every minute');
+	}
+}
+
+declare module '@sapphire/framework' {
+	interface ScheduledTasks {
+		interval: never;
+	}
+}
+```
+
+##### Using Interval tasks
+
+Cron & Interval tasks are loaded automatically.
+
+## Documentation
+
+For the full @sapphire/plugin-scheduled-tasks documentation please refer to the TypeDoc generated [documentation](https://sapphiredev.github.io/plugins/modules/_sapphire_plugin_scheduled_tasks.html).
+
```

Comment:
```suggestion

```

We don't link it here anymore since it's on sapphirejs. I removed that in all stuff but it was left here because older PR.